### PR TITLE
Add a blog by Nishant S

### PR DESCRIPTION
**Pull request by NishSatish**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — APPROVED** [review]

File: _data/blogs.yml [hunk 1]
```diff
@@ -272,3 +272,10 @@
   title: GCI 2017 with FOSSASIA
   img: axirygit.jpg
   author_name: Eiffel Chong
+
+- url: https://nishantgci.wordpress.com/2018/01/02/my-journey-in-open-source/
+  title: My journey in the world of open-source programming
+  img: pp.jpeg
+  author_name: Nishant S.
+
+
```
Comment (dilpreetsio) [inline _data/blogs.yml]: Just remove this extra line and we are good to go.

**Comment by codedsun** [comment]
@NishSatish kindly follow the above reviews and make the changes!

**Comment by abishekvashok** [comment]
@NishSatish  pls rebase properly.

**Review by ParthS007 — APPROVED** [review]

**Comment by NishSatish** [comment]
The URL?

**Pull request merged**